procedures/katello/branch.md.erb

```diff
@@ -43,18 +58,22 @@
   - [ ] [katello](https://github.com/katello/katello)
 - [ ] Bump versions to <%= develop %>-master
   - [ ] [katello](https://github.com/katello/katello)
-- [ ] Create PR to foreman-infra to add a mapping between the current Foreman and Katello release branches. Also remove the oldest mapping from the config. See [this example](https://github.com/theforeman/foreman-infra/pull/1491).
+- [ ] Create PR to [jenkins-jobs](https://github.com/theforeman/jenkins-jobs) to add a mapping between the current Foreman and Katello release branches. Also remove the oldest mapping from the config. See [this example](https://github.com/theforeman/jenkins-jobs/pull/54).
+- [ ] Review the Foreman schedule and planning ([example](https://community.theforeman.org/t/foreman-2-5-schedule-and-planning/22219)) and note the date of the first scheduled release candidate.
+- [ ] Generate and post the release procedure, if not already posted:
+  - [ ] (Note that Katello uses dots instead of dashes for release candidates, e.g. 4.1.1.rc1 not 4.1.1-rc1)
+  - [ ] Make sure to run `./tools.rb setup-environment` first
+  - [ ] Run `./tools.rb procedure release katello <%= release %>.rc1 <rc_release_date>` in [tool_belt](https://github.com/theforeman/tool_belt)
```

Comment:
This command doesn't work for me, it must be with a dash. At least currently:
```console
$ ./tools.rb procedure release katello 4.1.1.rc1
ERROR: parameter 'full-version': Release must be in major.minor.patch or major.minor.patch-rcx, like 1.20.0 or 1.20.0-rc1

See: 'tools.rb procedure release --help'
```

You now also need to provide a release owner and release engineer. If those are the same as in the current document, you can use the variables in the context.

Reply:
Since Katello does use periods here and not dashes, we should either change to use dashes or update this tool to work with either.  Either way it's outside the scope of this PR to me, but it's probably worth doing.

Reply:
Ok, updated the version number regex to accept either dots or hyphens.

Reply:
It still doesn't work:
```console
$ ./tools.rb procedure release katello 4.2.rc1 2021-10-01
ERROR: parameter 'full-version': Release must be in major.minor.patch (like 1.20.0), major.minor.patch-rcx (like 1.20.0-rc1), or major.minor.patch.rcx (like 1.20.0.rc1)

See: 'tools.rb procedure release --help'
```

I tested this by running:
```console
./tools.rb procedure branch katello 4.2 2021-10-01 the-owner the-engineer
```

After you fix the version number, you should also pass in arguments for the release owner and engineer.

Reply:
@ekohl No idea what happened here.. Around July 7 I had edited the regex to allow dots in version numbers, but I can't find the commit anywhere.  I ended up doing it again from scratch.  Should be working now.  (I see you've already added the arguments for the release owner and engineer.)

Reply:
> The release in this command is a major.minor so you need to add .0.

As far as I understand this command has never accepted major.minor, only major.minor.patch?  I'm even more confused now.  I tested the original regex from https://github.com/theforeman/tool_belt/blob/04cccf7f3af6cb852ec659f3e919b04d9aca558a/lib/tool_belt/commands/procedure.rb#L36 and it accepts `4.3.1` and `4.3.1-rc1`, but not `4.3` or `4.3-rc1`.

Reply:
I see the `update` and `branch` command params do ask for release in x.y, though.